scripts/build-upgrade-image.sh

```diff
@@ -67,9 +67,21 @@ aptly repo create -distribution=bionic -component=delphix upgrade-repository
 aptly repo add upgrade-repository debs
 aptly publish repo -skip-signing upgrade-repository
 
+#
 # Include version information about this image.
-VERSION=$(dpkg -f "$(find debs/ -name 'delphix-entire-*' | head -n 1)" version)
-sed "s/@@VERSION@@/$VERSION/" <version.info.template >version.info
+# If the virtualization package does not exist, then we will use the delphix-entire
+# package version as the "product" version.
+#
+ENTIRE_VERSION=$(dpkg -f "$(find debs/ -name 'delphix-entire-*' | head -n 1)" version)
+virt_pkg=$(find debs/ -name 'delphix-virtualization-*' | head -n 1)
+if [[ "$virt_pkg" ]]; then
+	VERSION=$(dpkg -f "$virt_pkg" version)
+else
+	VERSION=$ENTIRE_VERSION
+fi
+cp version.info.template version.info
+sed -i "s/@@ENTIRE_VERSION@@/$ENTIRE_VERSION/" version.info
```

Comment:
I think we should discuss things.. it strikes me as awkward how we now have `VERSION` which maps to the virtualization package's version.. and then `ENTIRE_VERSION` which maps to the entire package.

I don't really like how we're leaking more knowledge of the virtualization package, into the upgrade process/image. At this point, I feel like we should be more explicit and rename `VERSION` to `VIRTUALIZATION_VERSION`, and perhaps even include other package versions like masking, zfs, etc.. but, I'd rather not have any of these package versions at all, if we can avoid it.